airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1984,7 +1984,7 @@ paths:
     post:
```

Comment:
I think we likely will at some point, for now I wanted to keep the change non-breaking and backwards compatible with front-end, but once this lands we could discuss moving to `patch` instead of `post`. Right now our server makes some assumptions about everything being `POST`, and it'd be a breaking change on the front-end that we'd need to coordinate closely, but worth doing imo at some point in the near future.

 Some conversation about this in an issue that describes the broad convention for our API: https://github.com/airbytehq/airbyte/issues/13698